addons/product/models/product_template.py

```diff
@@ -949,16 +949,17 @@ def _is_combination_possible_by_config(self, combination, ignore_no_variant=Fals
         self.ensure_one()
 
         attribute_lines = self.valid_product_template_attribute_line_ids
+        has_multi_checkbox = any(a.display_type == 'multi' for a in attribute_lines.attribute_id)
 
         if ignore_no_variant:
             attribute_lines = attribute_lines._without_no_variant_attributes()
 
-        if len(combination) != len(attribute_lines):
+        if len(combination) != len(attribute_lines) and not has_multi_checkbox:
```

Comment:
```suggestion
        if len(combination_without_multi) != len(attribute_lines_without_multi):
```
to keep the same level of checks even if there are multi attribute lines ?